backend/seeders/20250502234154-student_events.js

```diff
@@ -0,0 +1,42 @@
+// eslint-disable-next-line no-unused-vars
+export async function up(queryInterface, Sequelize) {
+  queryInterface.bulkInsert(
+    'StudentEvents',
+    [
+      {
+        id: 1,
+        student_id: 'S12345',
+        meeting_id: 'M001',
+        event_action: 'joined',
+        createdAt: new Date('2025-04-20T08:00:00'),
+        updatedAt: new Date('2025-04-20T08:00:00'),
+      },
+      {
+        id: 2,
+        student_id: 'S67890',
+        meeting_id: 'M002',
+        event_action: 'left',
+        createdAt: new Date('2025-04-20T08:00:00'),
+        updatedAt: new Date('2025-04-20T08:00:00'),
+      },
+      {
+        id: 3,
+        student_id: 'S11223',
+        meeting_id: 'M003',
+        event_action: 'joined',
+        createdAt: new Date('2025-04-20T08:00:00'),
+        updatedAt: new Date('2025-04-20T08:00:00'),
+      },
```

Comment:
For this data, can you match the `meeting_id`'s and `student_id`'s with the existing IDs that are in the other seeder files? You can modify those seeder files if it helps